temporalnexus/operation.go

```diff
@@ -383,6 +387,7 @@ func ExecuteUntypedWorkflow[R any](
 	}
 	internal.SetLinksOnStartWorkflowOptions(&startWorkflowOptions, links)
 	internal.SetOnConflictOptionsOnStartWorkflowOptions(&startWorkflowOptions)
+	responseInfo := internal.SetResponseInfoOnStartWorkflowOptions(&startWorkflowOptions)
```

Comment:
Is there an end-to-end test to prove this works?

Reply:
Modified existing tests to check this.